### PR TITLE
Update Heroku pricing

### PR DESCRIPTION
**Pull request by CikiMomogi**
Heroku is no longer free as of November 28th, 2022.
See https://www.heroku.com/pricing for more.

### CODE REVIEW

File: site/docs/hosting/comparison.md [hunk 1]
```diff
@@ -29,21 +29,21 @@ We therefore recommend you to use a different kind of hosting for more complex b
...
+| Vercel Edge Functions  | Free       | $20/mo subscription for 500K                                                                               | [100K req/day](https://vercel.com/pricing)                                              | ❓       | ❓                           | ❓   |                                                                                                 |
+| serverless.com         | Free       |                                                                                                            |                                                                                         | ❓       | ❓                           | ❓   |                                                                                                 |
+| Heroku                 | $5         | $5 for 1,000 [dyno hours](https://devcenter.heroku.com/articles/usage-and-billing#dyno-usage-and-costs)/mo | [512MB RAM, sleeps after 30 mins of inactivity](https://www.heroku.com/pricing)         | ✅       | ✅                           | ❓   | Deno is supported by a [third-party buildpack](https://github.com/chibat/heroku-buildpack-deno) |
```
Comment (roj1512) [inline site/docs/hosting/comparison.md:44]:
```suggestion
| Heroku                 | $5         | $5 for 1,000 [dyno hours](https://devcenter.heroku.com/articles/usage-and-billing#dyno-usage-and-costs)/mo | [512MB RAM, sleeps after 30 mins of inactivity](https://www.heroku.com/pricing)         | ✅       | ✅                           | ❓   | Deno is supported by a [third-party buildpack](https://github.com/chibat/heroku-buildpack-deno). |
```

**Comment by roj1512** [comment]
Before we add the "ready for translation" label, @KnorpelSenf, did we say we will add a note or something like that back then when it was free?

**Comment by KnorpelSenf** [comment]
Yes, we should add a note to the Heroku guide and perhaps also add a note on the comparison page.

**Comment by CikiMomogi** [comment]
But why should we inform readers that Heroku's free subscription is no longer available? Can't we just remove them from docs?

**Comment by KnorpelSenf** [comment]
Do you want to remove it from both the comparison and from the hosting guides?

**Comment by CikiMomogi** [comment]
I am talking about directly replacing the free subscription with current pricing without any additional notes such as "Heroku free plan is no longer available." Perhaps you meant something else when you said "add a note"?

**Comment by KnorpelSenf** [comment]
Is this still active?

**Comment by roj1512** [comment]
Let me reactivate it.

**Review by habemuscode — APPROVED** [review]

**Comment by KnorpelSenf** [comment]
@roj1512 how's the translation guard supposed to work—shouldn't it prevent merges if a language is missing?

**Comment by CikiMomogi** [comment]
Hold on. What happened to commit https://github.com/grammyjs/website/pull/557/commits/677c6cf4a187ab1928b424054515005f4a60ffa4? I can't find the changes in the `files changed` tab.

Screenshot 2022-12-29 at 04-15-09 Update Heroku pricing by CikiMomogi · Pull Request #557 · grammyjs_website

**Comment by CikiMomogi** [comment]
Done. The Spanish doc now needs to get updated.

**Comment by roj1512** [comment]
@KnorpelSenf It currently requires only the ready for translation or not translatable, without looking for the language labels. Wanna change the behavior?

**Comment by KnorpelSenf** [comment]
> @KnorpelSenf It currently requires only the ready for translation or not translatable, without looking for the language labels.

Ahh right, makes sense

>Wanna change the behavior?

I believe it would be more intuitive, yes

**Comment by roj1512** [comment]
Works :)

**Comment by KnorpelSenf** [comment]
Awesome 😎

**Review by habemuscode — APPROVED** [review]

**Pull request merged**